```
$ eb --from-pr 6426 --check-style
== temporary log file in case of crash /tmp/eb-hhreSU/easybuild-DqqNmZ.log

*** WARNING: Using easyconfigs from unstable (pending/failed tests or merge conflict) PR #6426 ***

Running style check on 1 easyconfig(s)...
/tmp/eb-hhreSU/files_pr6426/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb:12:94: E226 missing whitespace around arithmetic operator
/tmp/eb-hhreSU/files_pr6426/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb:12:107: E226 missing whitespace around arithmetic operator
/tmp/eb-hhreSU/files_pr6426/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb:12:114: E226 missing whitespace around arithmetic operator
/tmp/eb-hhreSU/files_pr6426/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb:12:121: E501 line too long (137 > 120 characters)
/tmp/eb-hhreSU/files_pr6426/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb:12:127: E226 missing whitespace around arithmetic operator
[FAIL] /tmp/eb-hhreSU/files_pr6426/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb
ERROR: One or more style checks FAILED!
```

--- a/easybuild/easyconfigs/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb
+++ b/easybuild/easyconfigs/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb
@@ -1,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'libutempter'
+version = '1.1.6.2'
+
+homepage = 'http://git.altlinux.org/people/ldv/packages/?p=libutempter.git'
+description = """libutempter is library that provides an interface for terminal
+ emulators such as screen and xterm to record user sessions to utmp and wtmp files."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+source_urls = [
+    'http://git.altlinux.org/people/ldv/packages/?p=libutempter.git;a=snapshot;h='
+    + version[0:5] + '-alt' + version[6:7] + ';sf=tgz']
+sources = [{
+    'filename': '%s-%s-alt%s.tar.gz' % (name, version[0:5], version[6:7]),
+    'extract_cmd': "tar xfvz %s --strip 1",
+}]
+
+checksums = ['b7e55b666c575d8f2d294c82655b6af21f0c7339625ba6707e5edc2883a2b09a']
+
+skipsteps = ['configure']
+
+preinstallopts = 'sed -i "s#/usr##g" Makefile && '
+
+installopts = 'DESTDIR=%(installdir)s'
+
+sanity_check_paths = {
+    'files': ['include/utempter.h', 'lib/libutempter.%s' % SHLIB_EXT],
+    'dirs': ['include', 'lib', 'share/man/man3'],
+}
+
+moduleclass = 'devel'

